src/main/java/uk/ac/ebi/eva/pipeline/jobs/CommonJobStepInitialization.java

```diff
@@ -60,7 +66,22 @@ protected Step generateStep(String stepName, Tasklet tasklet) {
         return taskletBuilder.build();
     }
 
+    protected ObjectMap updateVariantOptionsWithJobSpecificValues (ObjectMap variantOptions){
```

Comment:
Sorry I explained myself pretty badly: my idea was to configure how samples are stored (INCLUDE_SAMPLES and COMPRESS_GENOTYPES) via `configureGenotypesStorage`, whereas the statistics storage (CALCULATE_STATS and INCLUDE_STATS) would be via `configureStatisticsStorage`. Those methods would then receive a couple of boolean arguments each, without being aware of which values each job class requires.